codes/classical/q-ary_digits/maximum_rank_distance.yml

```diff
@@ -0,0 +1,29 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: maximum_rank_distance
+physical: q-ary_digits
+logical: q-ary_digits
+
+name: 'Maximum rank distance code'
+introduced: '\cite{manual_citation_minilatex:{E. M. Gabidulin, "Theory of Codes with Maximum Rank Distance", Problemy Peredachi Informacii, Volume 21, Issue 1, \emph{3–16} (1985)}}'
+
+description: 'Linear error-correcting codes \([n,k,n-k+1]_q\) over rank metric'
+
+protection: '\(d\)-distance codes protect agains all errors of rank not greater \(\lfloor (d-1)/2\rfloor\)'
+
+notes:
+ - |
+   For any vector \(g=(g_1, \ldots, g_n)\) with linearly independent elements \(g_i\)s over \(GF(q)\), 
+   generator matrix \( G = \begin{pmatrix} g_1^{[0]} & g_2^{[0]} & \ldots & g_n^{[0]} \\ g_1^{[1]} & g_2^{[1]} & \ldots & g_n^{[1]} \\ g_1^{[k-1]} & g_2^{[k-1]} & \ldots & g_n^{[k-1]} \end{pmatrix} \)
+   where \([i]:=q^i\) and \(k=n-d+1\) defines a maximum rank distance code.
+
+relations:
+  parents:
+    - code_id: gabidulin
+      detail: 'Linear rank codes with maximum rank'
+  cousins:
+    - code_id: reed_solomon
+      detail: 'Similar but over different metric'
```

Comment:
What do you mean by "similar"? What specific feature of the code or its ingredients remind you of RS?

Reply:
Add details